package.json

```diff
@@ -88,7 +88,7 @@
     "lint:styles": "stylelint \"{projects,feature-libs}/**/*.scss\"",
     "prettier": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\"",
     "prettier:fix": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\" --write",
-    "start": "env-cmd --no-override -e dev,b2c,$SPA_ENV nx serve storefrontapp --configuration=development",
+    "start": "env-cmd --no-override -e dev,b2b,cpq,$SPA_ENV nx serve storefrontapp --configuration=development",
     "start:b2b": "env-cmd --no-override -e dev,b2b,$SPA_ENV nx serve storefrontapp --configuration=development",
     "start:ci": "env-cmd --no-override -e ci,b2c,$SPA_ENV nx serve storefrontapp --configuration=development",
```

Comment:
@Jali-Khusi, please, add `nx test cpq-quote --code-coverage` to `test:libs` script after `nx test cdc --code-coverage`, otherwise the `cpq-quote` tests would be executed